DESCRIPTION

```diff
@@ -0,0 +1,18 @@
+Package: vitalsigns
+Title: A Utility Package for Conservation International's Vital Signs Project 
+Version: 0.1
+Authors@R: person("Thomas", "Johnson", email = "thomascjohnson@gmail.com",
```

Comment:
Sorry, you are right. It will autogenerate from the cre field. No need for a separate Maintainer field.